build/helper/codegen_helper.py

```diff
@@ -877,7 +909,7 @@ def get_enum_value_snippet(value):
         'is_string': True,
         'is_repeated_capability': True,
         'is_session_handle': False,
-        'library_method_call_snippet': 'channel_list_ctype',
+        'library_method_call_snippet': 'self._repeated_capability',
```

Comment:
There are certain "repeated capability" parameters that are special. The `channel_list` is one of them. Rather than being passed along as a function argument, they're stored in `self._repeated_capability`, so that's where we need to pull them from when calling down from the Session to the next layer.